PLATFORMS

```diff
@@ -0,0 +1 @@
+linux/amd64,linux/s390x,linux/ppc64le,linux/arm64
```

Comment:
Platform file Added

Reply:
I would prefer this file be added in `arbitrary-users-patch` and `arbitrary-users-patch/happy-path/`, and adjust the scripts accordingly. That way the build script can be run from the repository root, or from within the directories themselves.